ddl/db_change_test.go

```diff
@@ -1062,6 +1062,54 @@ func (s *testStateChangeSuite) TestParallelAlterModifyColumn(c *C) {
 	s.testControlParallelExecSQL(c, sql, sql, f)
 }
 
+func (s *testStateChangeSuite) TestParallelAlterModifyColumnWithData(c *C) {
```

Comment:
Can we add more tests about `concurrent alter table`? For example, random test.

Reply:
Could you give an example of this random test? I don't think it's appropriate to add it here.